pubsub/pstest_test.go

```diff
@@ -16,13 +16,17 @@ package pubsub_test
 
 import (
 	"context"
+	"errors"
 	"strconv"
 	"sync"
 	"testing"
 
 	"cloud.google.com/go/internal/testutil"
 	"cloud.google.com/go/pubsub"
 	"cloud.google.com/go/pubsub/pstest"
+
+	// pubSubWithQueueError "github.com/MarErm27/alicebob/pubsub"
```

Comment:
No comments should be pushed, only meaningful doclines

Reply:
fixed